integration/integrate-with-email.adoc

```diff
@@ -0,0 +1,18 @@
+[id="integrate-using-email"]
+= Integrating with email
+include::modules/common-attributes.adoc[]
+:context: integrate-with-email
+
+toc::[]
+
+[role="_abstract"]
+You can use email as a notification method, and you can forward alerts from the {product-title} to a standard email provider. This topic explains how to integrate the {product-title-short} platform with your email provider.
```

Comment:
Should be "from {product-title}" with no 'the'.

Reply:
> Should be "from {product-title}" with no 'the'.

@neilcar verify the change per your request.